static/js/util/profile_edit.js

```diff
@@ -71,7 +73,68 @@ export function boundDateField(key, label, error) {
     <span className="validation-error-text">{error}</span>
   </div>;
 }
+export function boundEducationMonthField(keySet, label) {
+  const {updateProfile, profile} = this.props;
+  let onChange = month => {
+    let clone = Object.assign({}, profile);
 
+    let current_value = _.get(clone, keySet);
+        // format as ISO-8601
+    if(month === null){
+      _.set(clone, keySet, null);
+    }else if (current_value){
+      _.set(clone, keySet, moment(current_value).set('month', month.value).format('YYYY-MM-DD'));
+    } else {
+      _.set(clone, keySet, moment().set('day', 1).set('month', month.value).format('YYYY-MM-DD'));
+    }
+    updateProfile(clone);
+  };
+  let month = _.get(profile, keySet)? moment(_.get(profile, keySet), "YYYY-MM-DD").month() : null;
+  const month_options = [
+    { value: 0, label: 'January'},
+    { value: 1, label: 'February'},
+    { value: 2, label: 'March'},
+    { value: 3, label: 'April'},
+    { value: 4, label: 'May'},
+    { value: 5, label: 'June'},
+    { value: 6, label: 'July'},
+    { value: 7, label: 'August'},
+    { value: 8, label: 'September'},
+    { value: 9, label: 'October'},
+    { value: 10, label: 'November'},
+    { value: 11, label: 'December'}
+  ];
+
+
+  return <Select
```

Comment:
Should the user be able to clear the month select field? If so we should handle it in the above logic.